README.md

```diff
@@ -10,7 +10,22 @@ First, you'll need to configure a database. Once this project is deployed, we'll
 
 ### Windows setup
 
-Coming Soon!
+## Chocolatey and Postgres (Needs Testing)
+1. [Install Choclatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+2. Add `C:\ProgramData\chocolatey\bin` to your paths (type path in search, click Edit environment varaibles for you account).
+3. In your cmd.exe (admin) terminal run `choco install postgresql --params '/Password:<assigned to postgres user>` (cmd.exe (admin) can be found by pressing `win + x`).
+4. Upgrade postgresql: `choco upgrade postgresql`.
+5. Add `C:\Program Files\PostgreSQL\10\bin` to your paths.
+6. Start postgres: `psql -U postgres` you will see postgres-# in the terminal.
+7. Create your user with `CREATE USER name;` The terminal will print CREATE ROLE.
+8. Alter your role with permission to create a database with `ALTER USER name WITH CREATEDB CREATEROLE;` The terminal will print ALTER ROLE
+9. Check everything is in order with `\du`. The terminal will print a table with Role name, Atrributes, and Member of columns. If your user is listed you are good to go.
+10. Quit with `\q` or `ctr + c`.
+11. Start psql with new username with `psql`. You should see `user=>`.
```

Comment:
I got `psql: FATAL: password authentication failed for user "aaron"` Your instructions didn't say how to set a password. If I try to press "enter" without giving a password I get `psql: fe_sendauth: no password supplied`

Reply:
Yes I had many issues with the password.  It seems it did not set when prompted by the terminal.  At first I thought maybe it was a typo, but now I don't know.

Reply:
"For anyone else hitting an issue where the password is not being read from the cli parameter, it turned out to be caused by the package alias for me - calling "choco install postgresql" failed, but "choco install postgresql10" works."

Found this in the comments on the docs https://chocolatey.org/packages/postgresql.

Reply:
Also, this is how I updated my password. https://stackoverflow.com/questions/18587710/change-reset-postgresql-user-password-on-windows-7